### PR TITLE
CAKE-5019 Hook up remaining tracking events

### PR DESCRIPTION
**Pull request by mklucsarits**
Tracking events for new two-screen dialog

### CODE REVIEW

File: src/components/Preferences.js [hunk 1]
```diff
@@ -110,7 +139,7 @@ class Preferences extends Component {
                 <div className={`${globalStyles.dialog} ${styles.dialog}`}>
                     <div className={styles.content}>
                         <h2 className={`${styles.heading} ${styles.preferencesHeading}`}>{content.preferencesHeadline}</h2>
-                        <div className={styles.preferencesDescription}>
+                        <div className={styles.preferencesDescription} onClick={(e) => this.clickDescription(e)}>
```
Comment (JoshuaRogan) [inline src/components/Preferences.js]: You could use an instance field here instead if you prefer.

File: src/components/PreferencesSection.js [hunk 2]
```diff
@@ -15,9 +15,27 @@ class PreferencesSection extends Component {
     }
 
     toggleIsExpanded() {
+        const { purpose, tracker } = this.props;
         const { isExpanded } = this.state;
         this.setState({ isExpanded: !isExpanded });
         this.forceUpdate();
+
+        switch (purpose.id) {
```
Comment (JoshuaRogan) [inline src/components/PreferencesSection.js]: Maybe turn this into a function `trackPurposeId(id, tracker)` as it's used twice 🤷‍♂

Comment (mklucsarits) [reply src/components/PreferencesSection.js]: The events/methods called for the two are different though.

Comment (JoshuaRogan) [reply src/components/PreferencesSection.js]: Ah okay. I didn't realize that

**Pull request merged**